transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/extensions/TransformationDblClickHandler.java

```diff
@@ -73,19 +73,17 @@ public boolean canHandle(AbstractCamelModelElement clickedNode) {
 	 * @return	the parameter value or null
 	 */
 	private String getEndpointParameter(String uri, String key) {
```

Comment:
this is used:
![image](https://cloud.githubusercontent.com/assets/1105127/24196502/4fceb964-0efe-11e7-8981-37574dc0d218.png)